video_demo/video_demo.py

```diff
@@ -1,10 +1,20 @@
 import ctypes
 import numpy as np
 import time
-import cv2
+from cv2 import *
 import sys
+from numba import jit
+from common import clock, draw_str, StatValue, image_clamp
 
-from common import clock, draw_str
+
+"""Function to perform OpenCV Unsharp Masking"""
+@jit("uint8[::](uint8[::],float64,float64)",cache=True,nogil=True)
+def uscv(image,weight,threshold):
```

Comment:
Its good to have elaborate names, for variables, functions, classes etc. You can replace uscv -> unsharp_mask_cv